src/node-red/index.njk

```diff
@@ -3,9 +3,56 @@ layout: page
 nohero: true
 meta:
     title: Start using Node-RED online for free
-description:
-    Node-RED's low-code brilliance hosted on FlowFuse cloud. Answer your questions about getting started with Node-RED including tips, best practices, and FAQs to using Node-RED for 
-    different use cases.
+    faq:
```

Comment:
Can eleventyComputed data return data structures like an array of hashes?

Reply:
As a first iteration I believe this is good though. Just wondering if you considered it

Reply:
I'll look into it, I did investigate if I could call the data from the json I previously had and couldn't find a way. I'm gonna merge as is, but if you can point an article or some source of info to improve it, it would be very helpful 😊